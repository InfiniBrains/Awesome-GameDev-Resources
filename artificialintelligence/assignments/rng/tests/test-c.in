3 5 10 20
