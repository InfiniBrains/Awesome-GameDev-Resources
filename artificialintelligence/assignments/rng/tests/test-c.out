10
20
16
15
20
